### ISSUE TITLE
(Accessibility) Expandable tile has 2 focus elements and should have 1

### ISSUE THREAD

**Issue by DusanMilko** [IBM/carbon-components-angular#1491]
Expandable tile has 2 focus elements and it should only have one.

![Screen Shot 2020-09-03 at 5 34 48 PM](https://user-images.githubusercontent.com/302239/92180096-d3948b80-ee0b-11ea-8bce-90424dc5d44a.png)

Should match vanilla js and react html structure

![Screen Shot 2020-09-03 at 5 37 48 PM](https://user-images.githubusercontent.com/302239/92180285-4b62b600-ee0c-11ea-99eb-1ef88561c5ed.png)
